### DISCUSSION TITLE
2006/11/10/%E5%A6%82%E4%BD%95%E6%9C%89%E6%95%88%E5%9C%B0%E5%9C%A8%E9%9D%A2%E8%AF%95%E4%B8%AD%E8%80%83%E5%AF%9F%E5%BA%94%E8%81%98%E8%80%85%E7%9A%84%E8%83%BD%E5%8A%9B

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#128] [Announcements]
# 2006/11/10/%E5%A6%82%E4%BD%95%E6%9C%89%E6%95%88%E5%9C%B0%E5%9C%A8%E9%9D%A2%E8%AF%95%E4%B8%AD%E8%80%83%E5%AF%9F%E5%BA%94%E8%81%98%E8%80%85%E7%9A%84%E8%83%BD%E5%8A%9B

又到一年的招聘季节了。这一年我又面试的不少人，也被别人面试了几次。所以再次总结一下自己的心得。

https://blog.qizhen.xyz/2006/11/10/%E5%A6%82%E4%BD%95%E6%9C%89%E6%95%88%E5%9C%B0%E5%9C%A8%E9%9D%A2%E8%AF%95%E4%B8%AD%E8%80%83%E5%AF%9F%E5%BA%94%E8%81%98%E8%80%85%E7%9A%84%E8%83%BD%E5%8A%9B

**Comment by ruanqizhen** [comment]
之前博客下的留言：

Unknown
2008年01月22日 16:06 编辑
面史不懂

回复
鑫
2007年04月29日 16:18 编辑
喝水喝水0…

回复
风铃
2007年04月19日 17:03 编辑
获益非浅,谢谢!

回复
Haiying
2007年04月17日 12:35 编辑
真好，个人的经历如此可贵，辛苦你了。在这里获益非浅，谢谢！

回复
chen
2007年01月26日 11:34 编辑
     如果面试的结果经常会给别人带来挫败感和失望，我是很喜欢面试别人的，这决不是经常受到别人面试的逆反心理造成的，比如做惯了学生，一旦有一天让他监考了，夹袄下面的“小”字一定会显现的，毕竟他们还很年轻。^_^
     面试别人是自我检查和成长的最好手段之一，如果你真的想提高自己，又非常认真的对待这个过程。

回复
emily
2006年11月13日 13:56 编辑
呵呵都是一些熟悉的朋友
大家在上面随便走动一下咯~

回复
Mavis
2006年11月13日 10:19 编辑
恩~~~是门学问~~~

回复
i
2006年11月11日 13:59 编辑
也经历过面试,但只是针对学生的
一直很紧张哪!每次都是,都是…郁闷了
 
从你这里,多少受益啦!以后,我再参加面试,一定要提前来看两眼才行~
 
p.s.个人的经验之谈总是比泛泛之言要好,辛苦了
